core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/SimpleReplicationStrategy.java

```diff
@@ -30,22 +29,21 @@
 @ThreadSafe
 class SimpleReplicationStrategy implements ReplicationStrategy {
 
-  private final int replicationFactor;
+  private final ReplicationFactor replicationFactor;
 
   SimpleReplicationStrategy(Map<String, String> replicationConfig) {
     this(extractReplicationFactor(replicationConfig));
   }
 
-  @VisibleForTesting
```

Comment:
Nit: you can keep this annotation, the constructor is still only exposed for tests